index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Notes Application</title>
+    <link rel="stylesheet" href="sanitize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>My Notes Application</h1>
+    </header>
+
+    <div class="main_container">
+        <div class="form_container">
+            <h2>Create a New Note</h2>
+            <form>
+                <label>Title:</label>
```

Comment:
Implement `for` attribute, it will ensure `label` and `input` a11y (accessibility).